lambdas/service/app.py

```diff
@@ -2233,7 +2287,50 @@ def get_data_object(file_uuid):
         return Response(response.text, status_code=response.status_code)
 
 
-@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True)
+@app.route(drs.drs_http_object_path('{file_uuid}', access_id='{access_id}'), methods=['GET'], cors=True, method_spec={
+    'summary': 'Get a file with an access ID',
+    'description': format_description('''
+        This endpoint returns a URL that can be used to fetch the bytes of a DRS
+        object.
+
+        This method only needs to be called when using an `AccessMethod` that
+        contains an `access_id`
+    ''') + drs_description,
+    'parameters': [
+        params.path('file_uuid', str, description='UUID of the file to be fetched'),
+        params.path('access_id', str, description='Access ID returned from a previous request'),
+    ],
+    'responses': {
+        '301': {
```

Comment:
I'm confused. I thought the DRS standard doesn't have a place for 301s and 302s. Are we not implementing DRS correctly?

Reply:
You're right. The implementation is correct, but I screwed up on the documentation.

Reply:
This was a pretty gaping mistake in the spec... I've looked over the code carefully now and I believe it is correct.